README.md

```diff
@@ -1,13 +1,4 @@
 # CDK Construct library for building ECS services
-<!--BEGIN STABILITY BANNER-->
-
----
-
-![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
-
----
-
-<!--END STABILITY BANNER-->
```

Comment:
We should add this back in: showing raw and rendered verison:

```md
<!--BEGIN STABILITY BANNER-->

---

![cdk-constructs: Stable](https://img.shields.io/badge/cdk--constructs-stable-success.svg?style=for-the-badge)

---

<!--END STABILITY BANNER-->
```

<!--BEGIN STABILITY BANNER-->

---

![cdk-constructs: Stable](https://img.shields.io/badge/cdk--constructs-stable-success.svg?style=for-the-badge)

---

<!--END STABILITY BANNER-->